### PR TITLE
openstack-crowbar/ardana: rename engcloud/susecloud to ECP/ECN

### PR DESCRIPTION
**Pull request by stefannica**
Rename the engcloud/susecloud names currently used for the Provo and
Nuremberg engineering cloud instances to ECP and ECN respectively.

NOTE: this change also requires renaming the Jenkins Lockable Resources
and their associated labels, as well as updating the OpenStack cloud
configuration in `~/.config/openstack/cloud.yml` on Jenkins agent nodes.

### CODE REVIEW

File: jenkins/ci.suse.de/openstack-ardana-update-and-test.yaml [hunk 1]
```diff
@@ -48,14 +48,14 @@
...
-            - 'susecloud'
+            - 'ecp'
+            - 'ecn'
           description: >-
             The OpenStack cloud platform where the virtual cloud environment is
             running. Possible values are:
 
-              engcloud  - the Provo engineering cloud (engcloud.prv.suse.net)
-              susecloud - the Nuremberg SUSE cloud (cloud.suse.de)
+              ecp  - the Provo engineering cloud (engcloud.prv.suse.net)
```
Comment (rtamalin) [inline jenkins/ci.suse.de/openstack-ardana-update-and-test.yaml]: remove a space be fore the '-' to align them again

File: jenkins/ci.suse.de/openstack-ardana-tests.yaml [hunk 2]
```diff
@@ -48,14 +48,14 @@
...
-            - 'susecloud'
+            - 'ecp'
+            - 'ecn'
           description: >-
             The OpenStack cloud platform where the virtual cloud environment is
             running. Possible values are:
 
-              engcloud  - the Provo engineering cloud (engcloud.prv.suse.net)
-              susecloud - the Nuremberg SUSE cloud (cloud.suse.de)
+              ecp  - the Provo engineering cloud (engcloud.prv.suse.net)
```
Comment (rtamalin) [inline jenkins/ci.suse.de/openstack-ardana-tests.yaml]: Remove a space before the '-' to align them again.

File: jenkins/ci.suse.de/openstack-ardana-update.yaml [hunk 3]
```diff
@@ -48,14 +48,14 @@
...
-            - 'susecloud'
+            - 'ecp'
+            - 'ecn'
           description: >-
             The OpenStack cloud platform where the virtual cloud environment is
             running. Possible values are:
 
-              engcloud  - the Provo engineering cloud (engcloud.prv.suse.net)
-              susecloud - the Nuremberg SUSE cloud (cloud.suse.de)
+              ecp  - the Provo engineering cloud (engcloud.prv.suse.net)
```
Comment (rtamalin) [inline jenkins/ci.suse.de/openstack-ardana-update.yaml]: Remove a space before the '-' to align them again.

File: jenkins/ci.suse.de/templates/cloud-ardana-job-gerrit-template.yaml [hunk 4]
```diff
@@ -35,14 +35,14 @@
...
-            - 'susecloud'
+            - 'ecp'
+            - 'ecn'
           description: >-
             The target OpenStack cloud platform used as infrastructure for a
             virtual cloud deployment. Possible values are:
 
-              engcloud  - the Provo engineering cloud (engcloud.prv.suse.net)
-              susecloud - the Nuremberg SUSE cloud (cloud.suse.de)
+              ecp  - the Provo engineering cloud (engcloud.prv.suse.net)
```
Comment (rtamalin) [inline jenkins/ci.suse.de/templates/cloud-ardana-job-gerrit-template.yaml]: Remove a space before the '-' to align them again.

File: jenkins/ci.suse.de/templates/cloud-ardana-pipeline-template.yaml [hunk 5]
```diff
@@ -34,14 +34,14 @@
...
-            - 'susecloud'
+            - 'ecp'
+            - 'ecn'
           description: >-
             The target OpenStack cloud platform used as infrastructure for a
             virtual cloud deployment. Possible values are:
 
-              engcloud  - the Provo engineering cloud (engcloud.prv.suse.net)
-              susecloud - the Nuremberg SUSE cloud (cloud.suse.de)
+              ecp  - the Provo engineering cloud (engcloud.prv.suse.net)
```
Comment (rtamalin) [inline jenkins/ci.suse.de/templates/cloud-ardana-pipeline-template.yaml]: Remove a space before the '-' to align them again.

File: jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml [hunk 6]
```diff
@@ -34,14 +34,14 @@
...
-            - 'susecloud'
+            - 'ecp'
+            - 'ecn'
           description: >-
             The target OpenStack cloud platform used as infrastructure for a
             virtual cloud deployment. Possible values are:
 
-              engcloud  - the Provo engineering cloud (engcloud.prv.suse.net)
-              susecloud - the Nuremberg SUSE cloud (cloud.suse.de)
+              ecp  - the Provo engineering cloud (engcloud.prv.suse.net)
```
Comment (rtamalin) [inline jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml]: Remove a space before the '-' to align them again.

File: jenkins/ci.suse.de/templates/cloud-heat-template.yaml [hunk 7]
```diff
@@ -95,8 +95,8 @@
           description: >-
             The target OpenStack cloud platform. Possible values are:
 
-              engcloud  - the Provo engineering cloud (engcloud.prv.suse.net)
-              susecloud - the Nuremberg SUSE cloud (cloud.suse.de)
+              ecp  - the Provo engineering cloud (engcloud.prv.suse.net)
```
Comment (rtamalin) [inline jenkins/ci.suse.de/templates/cloud-heat-template.yaml]: Remove a space before the '-' to align them again.

File: jenkins/ci.suse.de/templates/cloud-image-update-template.yaml [hunk 8]
```diff
@@ -79,8 +79,8 @@
           description: >-
             The target OpenStack cloud platform. Possible values are:
 
-              engcloud  - the Provo engineering cloud (engcloud.prv.suse.net)
-              susecloud - the Nuremberg SUSE cloud (cloud.suse.de)
+              ecp  - the Provo engineering cloud (engcloud.prv.suse.net)
```
Comment (rtamalin) [inline jenkins/ci.suse.de/templates/cloud-image-update-template.yaml]: Remove a space before the '-' to align them again.

**Comment by dirkmueller** [comment]
would there be a problem with just using the hostname (engcloud.prv.suse.net)? as far as I can see this is just replacing one cryptic acronym with another one, not sure which one is better really (and in case we'll have at some point a duplicate for P or N in ECP/ECN, we'll just have to rename all of it again). 

I'd just use the engcloud.prv.suse.net vs cloud.suse.de instead.

**Comment by rsalevsky** [comment]
@dirkmueller I'm not really a fan of engcloudprv.suse.net vs. cloud.suse.de. In my opinion the url should be similar for both clouds. E.g. see https://jira.suse.com/browse/SOC-9853.

**Comment by dirkmueller** [comment]
@rsalevsky I think it really goes into the wrong direction. three letter acronyms are not a good choice. we could do something like engcloud.prv.suse.net or engcloud.nue.suse.net to align this, but thats not the discussion here. 

My feedback was "lets name it like the hostname are". and you're saying "lets change the hostnames to match the names in this pull request". I think thats the opposite of how this would work ;)

**Comment by rsalevsky** [comment]
Let's put this on hold till we have a decision in the jira card.

**Pull request closed without merging**